csrc/serde/fusion_record_serde.cpp

```diff
@@ -871,6 +871,7 @@ void RecordFunctorFactory::setupFunctionMaps() {
   NVFUSER_UNARY_TV_OP("reciprocal", reciprocal)
   NVFUSER_UNARY_TV_OP("round", round)
   NVFUSER_UNARY_TV_OP("rsqrt", rsqrt)
+  NVFUSER_UNARY_TV_OP("segment_set", segment_set)
```

Comment:
This isn't ideal... we shouldn't replicate this everywhere... I'll try to clean it up with a macro. :smiling_imp: